apps/agora/app/project.json

```diff
@@ -39,10 +39,15 @@
             "glob": "**/*",
             "output": "agora-assets"
           },
+          {
+            "input": "libs/explorers/assets",
+            "glob": "**/*",
+            "output": "explorers-assets"
+          },
           {
             "input": "libs/agora/assets/favicon",
-            "glob": "favicon.ico",
-            "output": ""
+            "glob": "**/*",
+            "output": "/"
```

Comment:
legacy code directly referenced just the favicon.ico.  We need all favicon assets in this folder now.